mne/io/brainvision/tests/data/test_highpass_hz.vhdr

```diff
@@ -0,0 +1,103 @@
+Brain Vision Data Exchange Header File Version 1.0
+; Data created by the Vision Recorder
```

Comment:
Hmmm... there might be a more efficient way with `difflib`, but I don't know if that helps with readability